### ISSUE TITLE
Create the Web App Logo

### ISSUE THREAD

**Issue by ADRE9** [ADRE9/bunk-manager-mern#94]
**Deliverables:**
Add the deliverables with their description here.
- [x] Design the Logo and ask for review.
- [x] Integrate the logo along with the favicon in the codebase

**Comment by himanshu007-creator** [comment]
Hi, i would like to work on this issue🙂. Please assign me this.

**Comment by ADRE9** [comment]
Yes go ahead. And also add the favicon for the app and edit the manifest.json file accordingly to show in the splash screen(for mobile) and in the tabs(for desktops)

**Comment by himanshu007-creator** [comment]
> Yes go ahead. And also add the favicon for the app and edit the manifest.json file accordingly to show in the splash screen(for mobile) and in the tabs(for desktops)

can you guide me later on regarding this?

**Comment by ADRE9** [comment]
> > Yes go ahead. And also add the favicon for the app and edit the manifest.json file accordingly to show in the splash screen(for mobile) and in the tabs(for desktops)
> 
> can you guide me later on regarding this?
> 

Sure! You can ask any mentors . If no one responds ,I'll. So don't worry.

**Comment by himanshu007-creator** [comment]
Hi, here are some suggestions, they seem vicious and funny at same time! 😂
![11](https://user-images.githubusercontent.com/65963997/113485113-a86d3a00-94c9-11eb-9823-2344e86d2b04.jpg)
![124](https://user-images.githubusercontent.com/65963997/113485116-a99e6700-94c9-11eb-8046-927044eff54e.jpg)
![12](https://user-images.githubusercontent.com/65963997/113485117-aacf9400-94c9-11eb-86ed-40b41643e218.jpg)
![125](https://user-images.githubusercontent.com/65963997/113485119-ac995780-94c9-11eb-95de-dc69953de615.jpg)
![126](https://user-images.githubusercontent.com/65963997/113485120-adca8480-94c9-11eb-9838-9c5841b4a1a3.jpg)

**Comment by ADRE9** [comment]
We want something professional.

**Comment by ADRE9** [comment]
We want something professional.

**Comment by ADRE9** [comment]
Hey there @himanshu007-creator  it has been 3 days already. What's the status of your work? I mean if you are not working on it we need to open it for participants and reassign it. Hoping that you come up with your PR soon.

**Comment by himanshu007-creator** [comment]
> Hey there @himanshu007-creator it has been 3 days already. What's the status of your work? I mean if you are not working on it we need to open it for participants and reassign it. Hoping that you come up with your PR soon.

No sir, I am working on it

**Comment by ADRE9** [comment]
So were you trying to check whether we are active or not. Because I guess I have got the best team in Girlscript. Mentors are super awesome and the participants are pretty good. We are trying to be easy on them. Btw it took me time to recognise you. But since you have been assigned to this job. Waiting for your PR. Else you can step back for other participants to take this assignment. 😎.

**Comment by himanshu007-creator** [comment]
> So we're you trying to check whether we are active or not. Because I guess I have got the best team in Girlscript. Mentors are super awesome and the participants are pretty good. We are trying to be easy on them. Btw it took me time to recognise you. But since you have been assigned to this job. Waiting for your PR. Else you can step back for other participants to take this assignment. 😎.

Cool, let someone else work.

**Comment by vtejaeta** [comment]
Can I work on this as a part of GSSOC. Please review the below one which I made in Figma.

![bunk manager](https://user-images.githubusercontent.com/65386350/114166547-7df90200-994b-11eb-8ca3-b5705aa639df.jpg)

**Comment by ADRE9** [comment]
Sure, go ahead. Make a minimal looking logo.

**Comment by vtejaeta** [comment]
![64x64_bunk_manager](https://user-images.githubusercontent.com/65386350/114268028-ae5ca100-9a1c-11eb-8ae6-13e4c47151bb.png)

This is 64x64 logo which can be used as favicon, If you like it I can add it as favicon in the code.

For SVG - 
https://svgshare.com/i/W0M.svg
